content/doc/developer/.htaccess

```diff
@@ -1 +0,0 @@
-ErrorDocument 404 /doc/developer/404/index.html
```

Comment:
Without a reference to the 404 file left, it should be removed as well.

Reply:
sorry i didnt get you what should be removed ?

Reply:
The file being referenced on this line.